app/models/filters/hmis_export.rb

```
@@ -282,5 +285,20 @@ def chosen(key)
         return super
       end
     end
+
+    def available_custom_file_types
+      return {} unless version == '2026'
+
+      begin
+        HmisCsvTwentyTwentySix.custom_files_config.definitions.map(&:for_select).to_h
+      rescue StandardError => e
+        Rails.logger.warn "Failed to load available custom file types: #{e.message}"
```

is it correct to swallow the error here?

---

I _think_ so.  My thinking was that we didn't want exports to fail if we were missing some yaml files, but it seems unlikely we'd hit this.  I'll remove the rescue for now.